src/components/cohortStudent/index.js

```diff
@@ -0,0 +1,24 @@
+import ProfileCircle from "../profileCircle"
+import "./style.css"
+
+
+const CohortStudent = ({firstName, lastName}) => {
+
+    if(firstName.length > 0 && lastName.length > 0) {
```

Comment:
Done, changed to if lengths = 0 and added undefined and null conditions as well. Will return empty fragment if any are true, else it will render normally